DYDX-Bots/Large-Inactive-Balance/README.md

```diff
@@ -0,0 +1,47 @@
+# Large Inactive Balance in Safety Module.
+
+## Description
+
+This bot detect users with large Inactive balance for the next Epoch in dYdX Safety Module.
+
+> The bot can operate in two different modes, used to determine the threshold of a _large_ inactive balance.
+>
+> - `STATIC` mode refers to the bot using a static predefined threshold.
+> - `PERCENTAGE` refers to setting the threshold as a percentage of the total staked tokens.
+
+> In order to switch between the two modes, change `DYNAMIC_CONFIG` to `STATIC_CONFG` in agent.ts, L60.
```

Comment:
If all the user is supposed to do is change `DYANMIC_CONFIG` to `STATIC_CONFIG` in `agent.ts`, `STATIC_CONFIG` also needs to be imported into `agent.ts`.